### PR TITLE
[Feature] Cache manager API 

### PR DESCRIPTION
**Pull request by rldhont**
This feature introduces a new QGIS Server API to get information about WMTS disk cache and to delete disk cache content

The available URLs are:
* /cachemngr.json
  * to get information on the WMTS disk cache
* /cachemngr/collections.json
  * to get the list of collections, QGIS projects, that have WMTS disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?).json
  * to get information on a collection, QGIS project, WMTS disk cache
  * to delete the collection, QGIS Project, WMTS disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/docs.json
  * to get information on a collection, QGIS project, WMTS documents disk cache
  * to delete the collection, QGIS Project, WMTS documents disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/layers.json
  * to get information on a collection, QGIS project, WMTS layers tiles disk cache
  * to delete the collection, QGIS Project, WMTS layers tiles disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/layers/(?<layerId>[^/]+?).json
  * to get information on a collection, QGIS project, WMTS layer tiles disk cache
  * to delete the collection, QGIS Project, WMTS layer tiles disk cache

To delete some cache, you have to use Delete HTTP method over the dedicated URL.

Funded by Ifremer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,5 +50,25 @@ Once installed, the `wmtscache` cache command can be used to manage the cache co
...
+* /cachemngr/collection/(?<collectionId>[^/]+?).json
+  * to get information on a collection, QGIS project, WMTS disk cache
+  * to delete the collection, QGIS Project, WMTS disk cache
+* /cachemngr/collection/(?<collectionId>[^/]+?)/docs.json
+  * to get information on a collection, QGIS project, WMTS documents disk cache
+  * to delete the collection, QGIS Project, WMTS documents disk cache
+* /cachemngr/collection/(?<collectionId>[^/]+?)/layers.json
+  * to get information on a collection, QGIS project, WMTS layers tiles disk cache
+  * to delete the collection, QGIS Project, WMTS layers tiles disk cache
+* /cachemngr/collection/(?<collectionId>[^/]+?)/layers/(?<layerId>[^/]+?).json
```
Comment (rldhont) [inline README.md:69]:
```suggestion
* /cachemngr/collections/(?<collectionId>[^/]+?).json
  * to get information on a collection, QGIS project, WMTS disk cache
  * to delete the collection, QGIS Project, WMTS disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/docs.json
  * to get information on a collection, QGIS project, WMTS documents disk cache
  * to delete the collection, QGIS Project, WMTS documents disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/layers.json
  * to get information on a collection, QGIS project, WMTS layers tiles disk cache
  * to delete the collection, QGIS Project, WMTS layers tiles disk cache
* /cachemngr/collections/(?<collectionId>[^/]+?)/layers/(?<layerId>[^/]+?).json
```
Fix the docs

**Comment by Gustry** [comment]
@rldhont I have rebased your branch on top of master.
You got a few conflicts after #5

**Comment by Gustry** [comment]
@rldhont @dmarteau I have also added a changelog.md file so it would need to edited before tagging to keep track.

**Comment by rldhont** [comment]
@dmarteau you have enhance this PR, do we merge it and you will update your branch and PR ?

**Comment by rldhont** [comment]
I close this one to merge #6

**Pull request closed without merging**